front-end/src/pages/main/components/MainCard.jsx

```
@@ -166,33 +166,37 @@ function renderTitle(recentSchedule, scheduleDifferece) {
-        <p className='body-1 text-center text-gray-400'>
-          나의 역할이 비어있어요
-        </p>
-        <MediumButton
-          text={'나의 역할 추가하기'}
-          isOutlined={false}
-          onClick={onButtonClick}
-        />
-      </div>;
+  try {
+    return (
+        recentSchedule.scheduleMemberNestedDtoList.find(
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)